src/core/seat/touch.cpp

```diff
@@ -427,24 +426,25 @@ static uint32_t find_swipe_edges(wf::touch::point_t point)
 {
     auto output   = wf::get_core().seat->get_active_output();
     auto geometry = output->get_layout_geometry();
+    wf::option_wrapper_t<int> edge_swipe_threshold{"input/edge_swipe_threshold"};
```

Comment:
This will cause a lookup for the option every time the function is called. I would make it a `static wf::option_wrapper_t<...>` so that it loads only the first time the function is called :)

Reply:
Thanks, will fix up after new year.